### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by YuikoTakada**
Update Japanese translation.

/hold for japanese developer's review
/assign @t-inu
/cc @atoato88 @s-kawamura-w664

### CODE REVIEW

**Review by t-inu — COMMENTED** [review]
Suggestions are only about spaces and parentheses.
I think the others are good.

**Comment by YuikoTakada** [comment]
@t-inu Thank you for your comments. I'll fix them.

File: translations/ja.json [hunk 1]
```diff
@@ -474,8 +474,8 @@
 	"Profile name '{{.profilename}}' is not valid": "プロファイル名 '{{.profilename}}' は無効です",
 	"Profile name should be unique": "プロファイル名は単一でなければなりません",
 	"Provide VM UUID to restore MAC address (hyperkit driver only)": "MAC アドレスを復元するための VM UUID を指定します (hyperkit ドライバーのみ)",
-	"Provides instructions to point your terminal's docker-cli to the Docker Engine inside minikube. (Useful for building docker images directly inside minikube)": "",
-	"Provides instructions to point your terminal's docker-cli to the Docker Engine inside minikube. (Useful for building docker images directly inside minikube)\n\nFor example, you can do all docker operations such as docker build, docker run, and docker ps directly on the docker inside minikube.\n\nNote: You need the docker-cli to be installed on your machine.\ndocker-cli install instructions: https://minikube.sigs.k8s.io/docs/tutorials/docker_desktop_replacement/#steps": "",
+	"Provides instructions to point your terminal's docker-cli to the Docker Engine inside minikube. (Useful for building docker images directly inside minikube)": "端末の docker-cli を minikube 内の Docker エンジンに指定する手順を提供します。（ minikube 内で直接 Docker イメージを構築するのに便利です）",
```
[the remark this answers is not in the document]
Comment (YuikoTakada) [reply translations/ja.json:477]: done

File: translations/ja.json [hunk 2]
```diff
@@ -426,11 +426,11 @@
...
 	"Overwrite image even if same image:tag name exists": "同じ image:tag 名が存在していてもイメージを上書きします",
 	"Path to the Dockerfile to use (optional)": "使用する Dockerfile へのパス (任意)",
-	"Path to the qemu firmware file. Defaults: For Linux, the default firmware location. For macOS, the brew installation location. For Windows, C:\\Program Files\\qemu\\share": "",
+	"Path to the qemu firmware file. Defaults: For Linux, the default firmware location. For macOS, the brew installation location. For Windows, C:\\Program Files\\qemu\\share": "qemuファームウェアファイルへのパス。デフォルト: Linux の場合、デフォルトのファームウェアの場所。macOS の場合、 brew のインストール場所。Windowsの場合、C:\\Program Files\\qemu\\share",
```
[the remark this answers is not in the document]
Comment (YuikoTakada) [reply translations/ja.json:433]: done

File: translations/ja.json [hunk 3]
```diff
@@ -474,8 +474,8 @@
...
+	"Provides instructions to point your terminal's docker-cli to the Docker Engine inside minikube. (Useful for building docker images directly inside minikube)": "端末の docker-cli を minikube 内の Docker エンジンに指定する手順を提供します。（ minikube 内で直接 Docker イメージを構築するのに便利です）",
+	"Provides instructions to point your terminal's docker-cli to the Docker Engine inside minikube. (Useful for building docker images directly inside minikube)\n\nFor example, you can do all docker operations such as docker build, docker run, and docker ps directly on the docker inside minikube.\n\nNote: You need the docker-cli to be installed on your machine.\ndocker-cli install instructions: https://minikube.sigs.k8s.io/docs/tutorials/docker_desktop_replacement/#steps": "端末の docker-cli を minikube 内の Docker エンジンに指定する手順を提供します。（ minikube 内で直接 Docker イメージを構築するのに便利です）\n\n例えば、docker build, docker run, docker ps などの全ての docker 操作を minikube 内の docker で直接実行できます。\n\n注意: docker-cli をマシンにインストールする必要があります。\ndocker-cli のインストール手順: https://minikube.sigs.k8s.io/docs/tutorials/docker_desktop_replacement/#steps",
```
[the remark this answers is not in the document]
Comment (YuikoTakada) [reply translations/ja.json:478]: done

File: translations/ja.json [hunk 4]
```diff
@@ -778,7 +778,7 @@
 	"User name must be 60 chars or less.": "ユーザー名は 60 文字以内でなければなりません。",
 	"Userspace file server is shutdown": "ユーザースペースのファイルサーバーが停止しました",
 	"Userspace file server: ": "ユーザースペースのファイルサーバー: ",
-	"Using Kubernetes v1.24+ with the Docker runtime requires cri-docker to be installed": "",
+	"Using Kubernetes v1.24+ with the Docker runtime requires cri-docker to be installed": "Docker ランタイムで Kubernetes v1.24+ を使用するには、cri-dockerをインストールする必要があります",
```
[the remark this answers is not in the document]
Comment (YuikoTakada) [reply translations/ja.json:781]: done

File: translations/ja.json [hunk 5]
```diff
@@ -426,11 +426,11 @@
...
 	"Overwrite image even if same image:tag name exists": "同じ image:tag 名が存在していてもイメージを上書きします",
 	"Path to the Dockerfile to use (optional)": "使用する Dockerfile へのパス (任意)",
-	"Path to the qemu firmware file. Defaults: For Linux, the default firmware location. For macOS, the brew installation location. For Windows, C:\\Program Files\\qemu\\share": "",
+	"Path to the qemu firmware file. Defaults: For Linux, the default firmware location. For macOS, the brew installation location. For Windows, C:\\Program Files\\qemu\\share": "qemu ファームウェアファイルへのパス。デフォルト: Linux の場合、デフォルトのファームウェアの場所。macOS の場合、 brew のインストール場所。Windows の場合、C:\\Program Files\\qemu\\share",
```
Comment (t-inu) [inline translations/ja.json:433]:
```suggestion
	"Path to the qemu firmware file. Defaults: For Linux, the default firmware location. For macOS, the brew installation location. For Windows, C:\\Program Files\\qemu\\share": "qemu ファームウェアファイルへのパス。デフォルト: Linux の場合、デフォルトのファームウェアの場所。macOS の場合、brew のインストール場所。Windows の場合、C:\\Program Files\\qemu\\share",
```
The space in front of "brew" that I suggested deleting is still remained.

Comment (YuikoTakada) [reply translations/ja.json]: done

**Review by t-inu — COMMENTED** [review]
/lgtm

**Comment by t-inu** [comment]
/assign @afbjorklund
Could you please merge this PR?

**Review by klaases — APPROVED** [review]
Thanks for your time - LGTM

**Review by spowelljr — APPROVED** [review]

**Pull request merged**